### PR TITLE
SEO-6 Per-wiki sitemap v1

### PR DESCRIPTION
**Pull request by gabrys**

### CODE REVIEW

File: extensions/wikia/GlobalFooter/GlobalFooterController.class.php [hunk 1]
```diff
@@ -41,15 +41,17 @@ private function getGlobalFooterLinks() {
...
-		if ( is_null( $globalFooterLinks = getMessageAsArray( self::MESSAGE_KEY_GLOBAL_FOOTER_LINKS . '-' . $verticalId ) ) ) {
-			if ( is_null( $globalFooterLinks = getMessageAsArray( self::MESSAGE_KEY_GLOBAL_FOOTER_LINKS ) ) ) {
-				wfProfileOut( __METHOD__ );
-				WikiaLogger::instance()->error(
-					"Global Footer's links not found in messages",
-					[ 'exception' => new Exception() ]
-				);
-				return [];
-			}
+		$globalFooterLinks = getMessageAsArray( self::MESSAGE_KEY_GLOBAL_FOOTER_LINKS, [
```
Comment (gabrys) [inline extensions/wikia/GlobalFooter/GlobalFooterController.class.php:42]: Removing the per-vertical messages as agreed with Chris F (not used anymore)

File: extensions/wikia/GlobalFooter/GlobalFooterController.class.php [hunk 2]
```diff
@@ -26,12 +26,12 @@ private function getGlobalFooterLinks() {
 
 		wfProfileIn( __METHOD__ );
 
-		$verticalId = WikiFactoryHub::getInstance()->getVerticalId( $wgCityId );
 		$memcKey = wfSharedMemcKey(
```
Comment (saipetch) [inline extensions/wikia/GlobalFooter/GlobalFooterController.class.php]: Consider using wfMemcKey ?

File: extensions/wikia/GlobalFooter/GlobalFooterController.class.php [hunk 3]
```diff
@@ -105,4 +107,17 @@ private function getVerticalShortName() {
 		}
 		return null;
 	}
+
+	private function getSitemapLink() {
+		global $wgCityId;
```
Comment (saipetch) [inline extensions/wikia/GlobalFooter/GlobalFooterController.class.php]: Consider using $this->wg->CityId?

**Comment by saipetch** [comment]
LGTM

**Pull request merged**